# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: modules/libpref/init/StaticPrefList.yaml
# Commit: 193184268c8e
# Full Hash: 193184268c8ef6347a77ef450d0ce008a7a360ee
# Author: longsonr <longsonr@gmail.com>
# Date: 2021-03-01 09:36:12
# Regressor Bug: 1600855
# File Overlap Count: 1
# Description:
#   Bug 1695490 - Remove svg.text-spacing.enabled pref r=emilio
#   
#   Backs out bug 1599173 which landed in Firefox 72. The pref has been enabled since bug 1600855 which landed in Firefox 73
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D106731
# ==============================================================================

diff -r 587d8a869c38 -r 193184268c8e modules/libpref/init/StaticPrefList.yaml
--- a/modules/libpref/init/StaticPrefList.yaml	Sun Feb 28 21:44:17 2021 +0000
+++ b/modules/libpref/init/StaticPrefList.yaml	Sun Feb 28 21:52:21 2021 +0000
@@ -9876,12 +9876,6 @@
   value: false
   mirror: always
 
-# Is support for letter-spacing and word-spacing in SVG text enabled?
-- name: svg.text-spacing.enabled
-  type: bool
-  value: true
-  mirror: always
-
 #---------------------------------------------------------------------------
 # Prefs starting with "telemetry."
 #---------------------------------------------------------------------------
